src/Screens/Gameplay/Components/Map/FocusedMap/FocusedMap.css

```diff
@@ -0,0 +1,10 @@
+.grid {
+	position: absolute;
+	margin: 0 auto;
+	left: calc(50% - calc(13 * var(--height-unit)));
```

Comment:
please do this to enhance readability -
![image](https://user-images.githubusercontent.com/67635802/116807806-e0ee4b00-ab3d-11eb-9554-1aecf82c2651.png)
and please add a comment to connect this data to the component via `styled-components`.